lighthouse/classes/events/biosero/destination_failed.py

```diff
@@ -49,8 +48,10 @@ def initialize_event(self, params: Dict[str, str]) -> None:
         self.properties["samples"] = SamplesFromDestination(self.properties["wells"])
         self.properties["samples_with_cog_uk_id"] = SamplesWithCogUkId(self.properties["samples"])
         self.properties["controls"] = ControlsFromDestination(self.properties["wells"])
-        self.properties["robot_serial_number"] = RobotSerialNumber(params)
-        self.properties["robot_uuid"] = RobotUUID(self.properties["robot_serial_number"])
+
```

Comment:
Why the space here?

Reply:
They are associated with run_info, do you want me to remove the whitespaces?